### PR TITLE
Add link read more in footer, and inform about changing networks

### PR DESCRIPTION
**Pull request by anxolin**
# Summary

## Before
image

## After

Add the read more link below the button:
image

also here:
image

Additionally, hint the user they might have a claiming available in another Network
image

  # To Test

1. Make sure the new links work

### CODE REVIEW

**Comment by biocom** [comment]
My first impression is, that the text link we use right below the primary button, is seen as a 'secondary' action. The 'Read more about vCOW' feels like it should be part of the description above instead.

**Comment by elena-zh** [comment]
Links work for me!
However, I'd add a network name in the description as we do when search for a token in token lists: 
image
image

**Review by elena-zh — APPROVED** [review]

**Review by elena-zh — CHANGES_REQUESTED** [review]
I noticed duplicating 'Read more' links on screens where Airdrop option is available to claim: 
image
It would be great to remove one of them.

File: src/custom/pages/Claim/CanUserClaimMessage.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ export default function CanUserClaimMessage({ hasClaims, isAirdropOnly, handleCh
           <ButtonSecondary onClick={handleChangeAccount} padding="0">
             Try another account
           </ButtonSecondary>{' '}
-          or <ExternalLink href={COW_LINKS.vCowPost}>read more about vCOW</ExternalLink>
+          or try in a different network.
```
Comment (alfetopito) [inline src/custom/pages/Claim/CanUserClaimMessage.tsx:46]: Agree with Elena, show what's the current network being checked

**Comment by anxolin** [comment]
Added the network name
image


Also remove the duplicated link

**Comment by anxolin** [comment]
Please review in the consolidated version :)

**Comment by elena-zh** [comment]
LGTM!

**Comment by W3stside** [comment]
Approved

**Comment by biocom** [comment]
Looks good. Two comments:
- We could later add a background color to the network name, using the same color as in the network switcher.
- Is it `...in {network}` or `...on {network}` ? For example: "Claims on Gnosis Chain".

**Pull request merged**